HULKs/hulk
# Update pose detection to yolo v11

## Why? What?

We experienced multiple issues with the old yolo pose detection results during RoboCup 2024. Some poses were not detected at all - arms straight up and either completely parallel or even pointing towards each other - or low background contrast to the person of interest resulted in bad confidence values and poses thus being rejected. 
This is an update from yolo v8 n pose to yolo v11 n pose. Initial tests resulted in substantially better detection of the challenging arms straight up pose as compared to the v8 version.   

||yolov8n-pose|yolov11n-pose|
|---|---|---|
|`ObjectDetectionTop.additional_outputs.inference_duration.millis`| ~270ms mean (with range 260 - 290ms) |~250ms mean (with range 220 - 270ms) |

Fixes #

## ToDo / Known Issues

*If this is a WIP describe which problems are to be fixed.*

## Ideas for Next Iterations (Not This PR)

*If there are some improvements that could be done in a next iteration, describe them here.*

## How to Test

- Setup a visual referee scenario
- Check all necessary poses for visual referee are detected and correctly filtered
